spec/terminal-tab-spec.js

```diff
@@ -7,12 +7,28 @@ import TerminalSession from '../lib/terminal-session';
 // To run a specific `it` or `describe` block add an `f` to the front (e.g. `fit`
 // or `fdescribe`). Remove the `f` to unfocus the block.
 
+function terminalOpenAndWait(workspaceElement) {
```

Comment:
I found this was a common utility so I factored it out. But not strongly opinionated on whether it's an embedded line in each test or not.